src/battle_anim_new.c

```diff
@@ -9148,3 +9148,22 @@ void AnimTask_GetWeatherToSet(u8 taskId)
     }
 	DestroyAnimVisualTask(taskId);
 }
+
+void AnimTask_SyrupBomb(u8 taskId)
+{
+    struct Pokemon *party = GetBattlerParty(gBattleAnimAttacker);
+    u32 isShiny = IsMonShiny(&party[gBattlerPartyIndexes[gBattleAnimAttacker]]);
+
+    gDisableStructs[gBattleAnimTarget].syrupBombIsShiny = isShiny;
```

Comment:
1. You should confirm this works in link battles because the disable struct is typically only used by the host GBA
2. If a second non-shiny Pokemon uses Syrup Bomb on the target when previously a shiny Pokemon had, the end turn animation will be affected even though the status won't have been reapplied.

Reply:
1. Seems fine (tested on mGBA)
![syrup_bomb_2](https://github.com/rh-hideout/pokeemerald-expansion/assets/93446519/59b06afd-4985-4fd3-b150-17fbb8b1abd2)
2. Will be fixed, thx :sweat_smile: